Source/WTF/wtf/Assertions.cpp

```diff
@@ -750,6 +750,12 @@ void WTFCrashWithInfoImpl(int, const char*, const char*, int, uint64_t reason)
     __builtin_unreachable();
 }
 
+void WTFCrashWithInfo(int, const char*, const char*, int)
+{
+    __asm__ volatile (CRASH_INST : : );
+    __builtin_unreachable();
```

Comment:
From talking to clang folks apparently we should use `__builtin_trap()` because `__builtin_unreachable()` is technically a signal to the compiler that this basic block can be removed.

Reply:
Sounds good. Changed.